emr-dynamodb-hadoop/src/main/java/org/apache/hadoop/dynamodb/DynamoDBClient.java

```diff
@@ -292,7 +292,12 @@ public BatchWriteItemResult call() throws
 
               double consumed = 0.0;
               for (ConsumedCapacity consumedCapacity : result.getConsumedCapacity()) {
-                consumed = consumedCapacity.getCapacityUnits();
+                consumed = consumedCapacity.getTable().getCapacityUnits();
+                if(null!=consumedCapacity.getLocalSecondaryIndexes()) {
```

Comment:
Minor: add whitespace around != to align the code style.

Reply:
, done synced with other null check format.